docs/middleware.md

```diff
@@ -107,6 +107,20 @@ The following arguments are supported:
 * `same_site` - SameSite flag prevents the browser from sending session cookie along with cross-site requests. Defaults to `'lax'`.
 * `https_only` - Indicate that Secure flag should be set (can be used with HTTPS only). Defaults to `False`.
 
+```python
+from starlette.applications import Starlette
+from starlette.middleware import Middleware
+from starlette.middleware.sessions import SessionMiddleware
+
+routes = ...
+
+middleware = [
+    Middleware(SessionMiddleware, https_only=True)
```

Comment:
I think you need the `secret_key` argument for `SessionMiddleware`

Reply:
@aminalaee you're right.

![image](https://github.com/encode/starlette/assets/4077212/f939daf6-9e23-4d5e-ae32-1f676cdef5b6)

I'm going to fix its now.. sorry. !!